src/cavacore.c

```diff
@@ -523,7 +524,7 @@ void cava_execute(double *cava_in, int new_samples, double *cava_out, struct cav
             p->sens_init = 0;
         } else {
             if (!silence) {
-                p->sens = p->sens * 1.002;
+                p->sens = p->sens * 1.01;
```

Comment:
Possible this change I didn't find during refactoring. Need to roll back once PR is approved